specification.swagger.yaml

```diff
@@ -965,57 +694,762 @@ paths:
-          description: Filter address by street. If a numeric identifier, the parameter is interpreted as a street resource identifier (id field). If not, it is interpreted as the exact street name, case insensitive.
+          description: Filter address by street. If a numeric identifier, the parameter is
+            interpreted as a street resource identifier (id field). If not, it
+            is interpreted as the exact street name, case insensitive.
+          schema:
+            type: string
+          example: siltasaarenkatu
         - name: number
           in: query
-          type: integer
           description: Filter by the number field of addresses.
+          schema:
+            type: integer
+          example: 1
       tags:
         - geography
       responses:
-        200:
+        "200":
           description: List of address resources
-          schema:
-            allOf:
-              - properties:
-                  results:
-                    type: array
-                    items:
-                      - $ref: '#/definitions/address'
-              - $ref: '#/definitions/common_collection'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - properties:
+                      results:
+                        type: array
+                        items:
+                          $ref: "#/components/schemas/address"
+                  - $ref: "#/components/schemas/common_collection"
+servers:
+  - url: http://localhost:8000/v2 #https://api.hel.fi/servicemap/v2
+components:
+  parameters:
+    page_param:
+      name: page
+      in: query
+      description: Request particular page in paginated results
+      required: false
+      schema:
+        type: integer
+      example: 1
+    pagesize_param:
+      name: page_size
+      in: query
+      description: Request that server delivers page_size results in response
+      required: false
+      schema:
+        type: integer
+      example: 10
+    include_param:
+      name: include
+      in: query
+      description: Embed the complete content of given reference-type fields directly into
+        the response, otherwise they are returned as identifiers. Separate field
+        names by commas.
+      schema:
+        type: array
+        items:
+          type: string
+    only_param:
+      name: only
+      in: query
+      description: Restrict the field returned in the results. Separate field names by
+        commas.'
+      schema:
+        type: array
+        items:
+          type: string
+    unit_bbox_param:
+      name: bbox
+      in: query
+      description: Search for events that are within this bounding box. Decimal coordinates
+        are given in order west, south, east, north. Period is used as decimal
+        separator.
+      schema:
+        type: array
+        items:
+          type: number
+      example: 382000.000,6673000.000,383000.000,6674000.000
```

Comment:
This gives Error 500 too.

Reply:
this doesn't work, because the example is used without a correct srid. In that case, api provides default srid, but it doesn't go together with given bbox coordinates. In validated version of smbackend this is solved with proper error messagies.